contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -447,6 +446,7 @@ var ContentNodeModel = BaseModel.extend({
         return attributes;
     },
     setExtraFields: function () {
+        const State = require("./state");
```

Comment:
can you explain this change?  why did you move away from a single import? to per-function imports?